mac

```diff
@@ -187,17 +191,6 @@ install_asdf_language() {
   fi
 }
 
-fancy_echo "Installing latest Ruby..."
-install_asdf_language "ruby"
-gem update --system
-gem_install_or_update "bundler"
-number_of_cores=$(sysctl -n hw.ncpu)
-bundle config --global jobs $((number_of_cores - 1))
-
-fancy_echo "Installing latest Node..."
-bash "$HOME/.asdf/plugins/nodejs/bin/import-release-team-keyring"
-install_asdf_language "nodejs"
```

Comment:
This code installed the latest version of Ruby and Node. We are usually not on the latest version, so I didn't think this was very helpful. I also think that not having any versions to begin with will give new hires a good opportunity to learn how to use asdf early on.

Reply:
However, being able to run node or ruby from the command line might be helpful. It also might help with setup instructions, for example, can't ruby install the necessary versions of itself using bundler? Or does everything always have a version manager independent of the code?

Reply:
I think that if someone wants to be able to do these things outside of an app repo, they should be responsible for picking the right version. I don't think we should assume it's always whatever the latest version happens to be.